test/integration/api/api_docs_controller_test.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Api::ApiDocsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Why do we have this and also https://github.com/3scale/porta/blob/master/test/integration/admin/api_docs/services_controller_test.rb?

Reply:
Because they are different controllers and different routes 😝 
For example, for the `index`, we have `Api::ApiDocsController` is the api_docs of a `service`, while `Admin::ApiDocs::ServicesController` is the api_docs of an `account`.

Reply:
In this case, shouldn't the `index` action be tested at https://github.com/3scale/porta/blob/7ad89feb35aae4643ca76c14431a263b64c4b524/test/integration/admin/api_docs/services_controller_test.rb#L18 as well?

Reply:
I get it now. This is really confusing IMO though.

Reply:
It's true it should have a test, but I didn't add it because the `Admin::ApiDocs::ServicesController#index` already existed before this PR and I didn't touch it 🤔 Although I touched the view but the view is exactly the same as I am testing here. Anyway, I didn't check if it had a test or not and I see it didn't 😱 so I will add it because it is very easy to add and should be there 😉

Reply:
Test pushed and documentation updated @guicassolato 👍 
In the documentation I only uploaded screenshots of everything, but it includes the URLs which may help 😄